Sources/SwiftUIBackports/Shared/Quicklook/Quicklook.swift

```diff
@@ -1,5 +1,5 @@
 import SwiftUI
-#if canImport(Quicklook)
+#if canImport(QuickLook)
 import QuickLook
```

Comment:
This would break all others I think given this was heavily tested. Plus I don't explicitly support or intend to support Catalyst with this library sorry.

Reply:
I don't quite understand how this would break the library. It just looks like there is a typo in the canImport statement. The "l" was lowercase in the canImport statement but in the line with the import it's capitalized.

Reply:
Oh I apologise you're right. I thought you'd renamed both. I was on the train and just skimmed before replying – totally makes sense 👍